composer.json

```diff
@@ -43,32 +43,32 @@
         "symfony/web-profiler-bundle": "^5.3",
         "symfony/webpack-encore-bundle": "^1.8",
         "symfony/yaml": "^5.3",
-        "symplify/amnesia": "9.4.22",
-        "symplify/autowire-array-parameter": "9.4.22",
-        "symplify/composer-json-manipulator": "9.4.22",
-        "symplify/easy-testing": "9.4.22",
-        "symplify/markdown-diff": "9.4.22",
-        "symplify/package-builder": "9.4.22",
-        "symplify/php-config-printer": "9.4.22",
-        "symplify/rule-doc-generator": "9.4.22",
-        "symplify/simple-php-doc-parser": "9.4.22",
-        "symplify/skipper": "9.4.22",
-        "symplify/smart-file-system": "9.4.22",
-        "symplify/symfony-php-config": "9.4.22",
-        "symplify/symplify-kernel": "9.4.22"
+        "symplify/amnesia": "^9.4.23",
+        "symplify/autowire-array-parameter": "^9.4.23",
+        "symplify/composer-json-manipulator": "^9.4.23",
+        "symplify/easy-testing": "^9.4.23",
+        "symplify/markdown-diff": "^9.4.23",
+        "symplify/package-builder": "^9.4.23",
+        "symplify/php-config-printer": "^9.4.23",
+        "symplify/rule-doc-generator": "^9.4.23",
+        "symplify/simple-php-doc-parser": "^9.4.23",
+        "symplify/skipper": "^9.4.23",
+        "symplify/smart-file-system": "^9.4.23",
+        "symplify/symfony-php-config": "^9.4.23",
+        "symplify/symplify-kernel": "^9.4.23"
     },
     "require-dev": {
-        "phpstan/phpstan": "0.12.91",
+        "phpstan/phpstan": "^0.12.91",
```

Comment:
using this to avoid overlap rector cannot update as it pinned phpstan 0.12.91 while latest rector uses phpstan 0.12.93

Reply:
I think we can go to 0.12.93 here. I've upgraded the Rector to use the latest PHPStan

Reply:
that's not work as current rector still uses phpstan 0.12.91, update to use `0.12.93` will get error:

```bash
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - rector/rector[0.11.36, ..., 0.11.38] require phpstan/phpstan 0.12.91 -> found phpstan/phpstan[0.12.91] but it conflicts with your root composer.json require (^0.12.93).
    - Root composer.json requires rector/rector ^0.11.36 -> satisfiable by rector/rector[0.11.36, 0.11.37, 0.11.38].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

Reply:
It was not released yet probably.
Could you tag rector if you have access? I'm not on PC now

Reply:
Ok, I just tagged 0.11.39 https://github.com/rectorphp/rector-src/releases/tag/0.11.39

Reply:
updated to use phpstan 0.12.93 and rector to 0.11.39